std/algorithm/searching.d

```diff
@@ -4027,6 +4027,64 @@ if (is(typeof(binaryFun!pred(r.front, e))) && isInputRange!R)
 }
 
 /**
+ * An overload of `skipOver` which takes a predicate. Skips over any elements
+ * from the start of the range that evaluate to `true`. Stops popping elements
+ * on the first element which `pred(r.front)` equals `false`.
+ *
+ * Params:
+ *     pred = the test for each element
+ *     r = the range to skip over
+ * Returns:
+ *     `true` if elements were skipped over, `false` otherwise.
+ */
+bool skipOver(alias pred, R)(ref R r) if (ifTestable!(typeof(r.front), unaryFun!pred))
```

Comment:
Nice. Please add but use `/// ditto` for the documentation and consolidate with the existing doc.